scripts/release_helper/python.py

```diff
@@ -41,10 +44,23 @@ def init_readme_link(self) -> None:
 
         # Get the origin link and readme tag in issue body
         origin_link, self.target_readme_tag = get_origin_link_and_tag(issue_body_list)
+        self.is_specified_tag = any('->Readme Tag:' in line for line in issue_body_list)
 
         # get readme_link
         self.get_readme_link(origin_link)
 
+    def check_tag_consistency(self) -> None:
+        self.target_readme_tag = self.target_readme_tag.replace('tag-', '')
+        super().check_tag_consistency()
+        if self.default_readme_tag != self.target_readme_tag:
+            self.add_label(_INCONSISTENT_TAG)
+
+    def multi_api_policy(self) -> None:
+        if self.is_multiapi:
+            self.bot_advice.append('It is a MultiAPI')
```

Comment:
```suggestion
            self.bot_advice.append('MultiAPI')
```

Reply:
If there is already `auto-ask-check`, it proves that we have handled it and then there is no need to hint it.